crt_portal/cts_forms/tests/integration/unsupported_browser.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+@pytest.fixture(scope="function")
+def browser_context_args(browser_context_args):
+    return {
+        **browser_context_args,
+        "userAgent": "SamsungBrowser/i"
```

Comment:
`/i` (case-insensitive regexp flag) is not necessary here, can we remove?

Reply:
... caught my copy/pasting 😂 

✔️